_contributors-app/arijeetsat.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Arijeet Satapathy
+description: I am a SDN(Software Defined Networking) Researcher and Machine Learning Enthusiast.
+picUrl: https://avatars0.githubusercontent.com/u/23520970?s=400&u=7831e0cae5f03ee34e50743d3d49e4b40c52461a&v=4%22
+skills: |
+  - **Languages:** Python, P4, MATLAB, C/C++
+  - **Frameworks:** POX , Floodlight, Pandas ,Numpy, OpenCV
+  - **Tools:** git, Github
+
+githubUsername: arijeetsat
+speciality: |
+
+  - Software Defined Networks
+
+  - Competitve Programming
+
+  - Machine Learning
+
+  - 
```

Comment:
Remove this extra `-`

Reply:
Also when you have removed the `-` and saved the file push it as a part of your existing commit don't make a new one. Follow these steps:
`git add .`
`git commit --amend --no-edit`
`git push -f <fork-remote-name> <branch-name>`